python/dolfinx/mesh.py

```diff
@@ -439,7 +440,20 @@ def create_mesh(
     return Mesh(mesh, domain)
 
 
-def create_submesh(msh, dim, entities):
+def create_submesh(msh: Mesh, dim: int, entities: npt.NDArray[np.int32]):
+    """
+    Create a mesh based on a subset of entities from an existing mesh.
```

Comment:
This should be one line up. Something must be missing in the ruff checks to not pick this up.